doc/usingchapel/bugs.rst

```diff
@@ -1,53 +1,24 @@
 .. _readme-bugs:
 
-=====================
-Reporting Chapel Bugs
-=====================
+=======================
+Reporting Chapel Issues
+=======================
 
-If you encounter a bug, internal error, or unclear error message when
-using Chapel, please do not hesitate to send a bug report to the
-Chapel team.  Do so by sending mail to one of the following mailing
-lists:
+Public Issues
+-------------
 
-  * chapel-bugs@lists.sourceforge.net : for bugs that are suitable for `public archive <https://sourceforge.net/p/chapel/mailman/chapel-bugs/>`_
-  * chapel_bugs@cray.com              : for bugs that the Chapel team at Cray Inc. should keep private
+Chapel uses `GitHub Issues`_ to track public issues. If you have a bug report,
+feature request, or other Chapel issue to report, please use the "New Issue"
```

Comment:
What do you think about dropping the second occurrence of "Chapel" just above?  Seems unnecessary and it has the unfortunate effect of lining up right under the previous Chapel in my browser...